### DISCUSSION TITLE
articles/relearn-c

### DISCUSSION THREAD

**Discussion by hutusi** [hutusi/hutusi.github.com#137] [Comments]
https://hutusi.com/articles/relearn-c 

   多年以后，面对考试题，我会想起开始学习谭浩强《C语言编程》的那个下午。   —-《百年C语言》

**Comment by liquid207** [comment]
终于，人类想起了被C语言支配的恐惧了。

**Comment by stevenqiulei** [comment]
谭浩强的《C语言编程》误了几千万中国学子。。

**Comment by hutusi** [comment]
@stevenqiulei 哈哈，不过谭对C语言的普及贡献还是有的，毕竟发行量那么大...
